debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -472,6 +472,8 @@ protected Object extractValueFromMessage(PgProto.DatumMessage datumMessage) {
                 PgProto.Point datumPoint = datumMessage.getDatumPoint();
                 return new PGpoint(datumPoint.getX(), datumPoint.getY());
             }
+            case PgOid.TSTZRANGE_OID:
+                return datumMessage.hasDatumBytes() ? new String(datumMessage.getDatumBytes().toByteArray()) : null;
```

Comment:
This `String` constructor relies on the default charset of the current platform. I'm not sure whether that's a potential source for trouble, could we thus use the constructor that takes the charset as an additional parameter?